Makefile

```diff
@@ -161,12 +163,25 @@ undeploy: kustomize delete-kubeconfig-cm
 	rm -rf ${RUNFILES}
 
 deploy-calico:
-	kubectl apply -f deploy/calico-with-grpc.yaml 
+	@if [ $(host_k8s_ver) = $(TARGET_K8S_VER) ]; then \
+		kubectl apply -f deploy/calico-with-grpc-3.26.1.yaml ;\
+	elif [ "$(TARGET_K8S_VER)" = "`echo -e "$(host_k8s_ver)\n$(TARGET_K8S_VER)" | sort -V | head -n1`" ] ; then \
```

Comment:
Why do you need this? Wouldn't runtime kubelet version be a good indicator?

Reply:
Yes. the host_k8s_ver is taking from the runtime kubelet version. Please check line number 37